crs-apache/Dockerfile

```
@@ -1,7 +1,7 @@
 ARG DOCKER_REGISTRY_URL
 ARG ONEC_VERSION
 
-FROM ${DOCKER_REGISTRY_URL}/crs:${ONEC_VERSION}
+FROM ${DOCKER_REGISTRY_URL:+"$DOCKER_REGISTRY_URL/"}crs:${ONEC_VERSION}
```

Hadolint (2.14.0)

[error] unexpected ':'
expecting '@', '\', a new line followed by the next instruction, or the image tag

(DL1000)